services/web/client/source/class/qxapp/components/workbench/Workbench.js

```diff
@@ -40,23 +42,40 @@ qx.Class.define("qxapp.components.workbench.Workbench", {
     this.__nodes = [];
     this.__links = [];
 
-    let plusButton = this.__getPlusButton();
+    let plusButton = this.__getPlusMenuButton();
     this.add(plusButton, {
-      right: 20,
+      right: 2*(50+10) + 20,
```

Comment:
Would it make sense to define some variable with these constants to make it clear what the intent is?